views/templates/admin/marketing_step0/marketing_step0.tpl

```diff
@@ -15,51 +15,51 @@
                     {if $smarty_email_promotion}
-                        <span class="badge"><i class="icon-star"></i> <small>{l s='Discount' mod='expressmailing'}</small></span>
+                        <br/><span class="badge"><i class="icon-star"></i> <small>{l s='Discount' mod='expressmailing'}</small></span>
                     {/if}
                 </div>
             </div>
         </div>
-        <div style="display: inline-block; width:24%; min-width: 180px;">
-            <img src="../modules/expressmailing/views/img/fax.png" border="0" style="display: inline-block">
-            <div style="display: inline-block; vertical-align: middle; padding-left: 1em; text-align: left">
+        <div style="display: inline-block; width:24%; min-width: 180px; text-align: left; vertical-align: top; margin-bottom: 1.5em">
+            <img src="../modules/expressmailing/views/img/fax.png" border="0" style="display: inline-block; margin-bottom: .5em">
+            <div style="display: inline-block; vertical-align: top; padding-left: 1em; text-align: left; width: 18em">
                 <span style="display: block; font-size: 10pt; font-weight: bold">Vos crédits fax</span>
                 <span style="display: block; font-size: 9pt">{$smarty_remaining_fax_credits|unescape}</span>
                 <div style="display: block; padding-top: 4px">
                     <a href="index.php?controller=AdminMarketingFList&token={Tools::getAdminTokenLite('AdminMarketingFList')|escape:'html':'UTF-8'}" class="btn btn-default" style="text-align: left; box-shadow: none; border: none;"><i class="icon-dashboard"></i> &nbsp;{l s='My fax stats' mod='expressmailing'}</a><br/>
                     <a class="btn btn-default" id="em_bying_link_fax" style="text-align: left; box-shadow: none; border: none;"><i class="icon-shopping-cart"></i> &nbsp;{l s='Buy tickets' mod='expressmailing'}</a>
                     {if $smarty_fax_promotion}
-                        <span class="badge"><i class="icon-star"></i> <small>{l s='Discount' mod='expressmailing'}</small></span>
+                        <br/><span class="badge"><i class="icon-star"></i> <small>{l s='Discount' mod='expressmailing'}</small></span>
                     {/if}
                 </div>
             </div>
         </div>
-        <div style="display: inline-block; width:24%; min-width: 180px;">
-            <img src="../modules/expressmailing/views/img/sms.png" border="0" style="display: inline-block">
-            <div style="display: inline-block; vertical-align: middle; padding-left: 1em; text-align: left">
+        <div style="display: inline-block; width:24%; min-width: 180px; text-align: left; vertical-align: top; margin-bottom: 1.5em">
+            <img src="../modules/expressmailing/views/img/sms.png" border="0" style="display: inline-block; margin-bottom: .5em">
+            <div style="display: inline-block; vertical-align: top; padding-left: 1em; text-align: left; width: 18em">
```

Comment:
As you can see, your CSS code is redundant. You can avoid this by moving it into a CSS file.